### ISSUE TITLE
Fetch last matches from the Excels

### ISSUE THREAD

**Issue by kelostrada** [kelostrada/chitczat#7]
Get all the company employees to database from Slack and later maybe manually add last matches to the database from excels?

**Comment by agusmoles** [comment]
I was going to start making an script to migrate those excel records to the db but I think first we should be all good set-up before migrating

**Comment by kelostrada** [comment]
Yeah, I believe we should at least refactor the matches so we have a proper structure. But part of this task is actually fetching all the users first and we could write the script maybe to pre-fetch all the slack users to our database so we have their IDs already prepared.

**Comment by agusmoles** [comment]
IMO there is no need to have all the users in the db, in the migration of previous paired people we will have those that usually joins the chitczat queue but then there is no need to have more people from the company if the don't

**Comment by kelostrada** [comment]
Well what I meant is to have those that used chitczat ever. Because they will be in the Excels and we need to match their Slack IDs with the pairs from Excels. Unless you have different idea on the approach :)
We probably want to fetch like last 10 excel tables though, might not be necessary to import all of them
